executor/aggfuncs/func_group_concat.go

```diff
@@ -280,6 +280,10 @@ type topNRows struct {
 	currSize  uint64
 	limitSize uint64
 	sepSize   uint64
+	// If (separator, row) pair is poped from heap, we need to append separator in concat()
+	// In the following example, (sep2, str3) pair is poped, but partial sep2 need to be appended.
+	// eg: "str1 sep1 str2 sep2 str3" -> "str1 sep1 str2 se"
```

Comment:
1. sep1 --> sep
2. sep2 --> sep
3. what's the group_concat_length of this example